### ISSUE TITLE
Pre-work Checklist: Developer: Tony Vang

### ISSUE THREAD

**Issue by tonyvang** [hackforla/website#5486]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [ ] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [ ] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [ ] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [ ] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [ ] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by roslynwythe** [comment]
Hi @tonyvang please click the gear icon to the right of "Projects" (under the Labels control) and select "Project Board".  Then apply the label "ready for dev lead".  Thank you.

**Comment by tonyvang** [comment]
Hi @roslynwythe, thanks for the comment; I've added this issue to the project board and applied the "ready for dev lead" label.

**Comment by tonyvang** [comment]
**Progress:** I am working on and have completed action items in the checklist prior to "Progress through issues... with increasing complexity..."; I have completed my first small issue, hope to do a few more and and will then try tackling a medium issue. I have not yet done steps re Figma familiarization.

**Blockers:** Life's been throwing car trouble at me this week and I've had a lot of my bandwidth taken up with that.

**Availability:** "I will set aside time to look at another issue/review pull requests on Monday and Tuesday"

**ETA:** "I hope to go into 2024 with all pre-work checklist items finished!

**Pictures:** from PR 5917:

<details>
  <summary>Visuals before changes are applied</summary>

![283664688-3150944a-ac71-49cf-9754-8bb9cb02803c](https://github.com/hackforla/website/assets/67331818/dc40f9d1-a644-47d6-961d-d95ee4445706)

</details>

<details>
  <summary>Visuals after changes are applied</summary>

![283664701-180fec57-769f-4174-8a9e-bb28a252a437](https://github.com/hackforla/website/assets/67331818/e08bde22-f669-4ad9-b1a7-6c4f88595d12)

</details>

**Comment by ExperimentsInHonesty** [comment]
@tonyvang Hi, I wanted to follow up with you, about your work on the website team and to encourage you to complete areas where you have not done any issues yet.  I have filled out this checklist based on the work I can see via the PR and Issue tab

- [x] Setting up your local environment from a contributing file
- [x] GitHub branching
- [x] Pull Requests
- [x] Reviewed other people's Pull Requests
- [ ] Resolved Merge Conflicts
- [ ] Written documentation for other Developers (Architecture, etc.)
- [ ] Mentored other developers
- [ ] Making issues for other developers to work on
- [ ] API requests
- [ ] Cron Job Scripting (GitHub Actions)
- [ ] CRUD operations

### Next issue for you to work on
- [medium issues in the prioritized backlog](https://github.com/hackforla/website/projects/7?card_filter_query=-label%3A%22role%3A+user+research%22+-label%3A%22role%3A+dev+leads%22+-label%3A%22role%3A+design%22+label%3A%22complexity%3A+medium%22#column-7198257)

**Comment by tonyvang** [comment]
@ExperimentsInHonesty thanks for following up; a new job I started in January and another regular commitment have picked up, and I've been having difficulty setting aside the proper time for H4LA. I had hoped to work up to the point where I could properly contribute through working/reviewing medium and large issues, but I'm still at a point in my experience and learning where the ones I've seen in the Prioritized Backlog in the past few weeks have been daunting. Once the couple of PRs I'm reviewing this week have merged or closed, I may message Roslyn and deactivate for the time being, rather than take up more of the team's time and training resources.

**Comment by HackforLABot** [comment]
The Hack for LA Bot has closed this issue due to member inactivity.

**Comment by HackforLABot** [comment]
## Activity Log: tonyvang
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- tonyvang opened: https://github.com/hackforla/website/issues/5486 at 09/11/2023, 08:33 PM PDT
- tonyvang assigned: https://github.com/hackforla/website/issues/5486 at 09/11/2023, 08:33 PM PDT
- tonyvang commented on: https://github.com/hackforla/website/issues/5486#issuecomment-1716366461 at 09/12/2023, 01:18 PM PDT
- tonyvang assigned: https://github.com/hackforla/website/issues/5508 at 09/21/2023, 06:53 PM PDT
- tonyvang commented on: https://github.com/hackforla/website/issues/5508#issuecomment-1731743565 at 09/22/2023, 09:56 AM PDT
- tonyvang commented on: https://github.com/hackforla/website/issues/5508#issuecomment-1732235751 at 09/22/2023, 11:59 PM PDT
- tonyvang commented on: https://github.com/hackforla/website/issues/5508#issuecomment-1732350400 at 09/23/2023, 09:06 AM PDT
- tonyvang opened: https://github.com/hackforla/website/pull/5576 at 09/23/2023, 10:42 PM PDT
- tonyvang commented on: https://github.com/hackforla/website/pull/5576#issuecomment-1737874513 at 09/27/2023, 11:21 AM PDT
- tonyvang PR was merged: https://github.com/hackforla/website/pull/5576#event-10512430444 at 09/29/2023, 07:57 AM PDT
- tonyvang issue closed by PR 5576: https://github.com/hackforla/website/issues/5508#event-10512430685 at 09/29/2023, 07:57 AM PDT
- tonyvang commented on: https://github.com/hackforla/website/pull/5701#issuecomment-1756713503 at 10/10/2023, 08:36 PM PDT
- tonyvang submitted review: https://github.com/hackforla/website/pull/5701#pullrequestreview-1678829800 at 10/15/2023, 09:53 AM PDT
- tonyvang assigned: https://github.com/hackforla/website/issues/5753 at 10/24/2023, 03:40 PM PDT
- tonyvang commented on: https://github.com/hackforla/website/issues/5753#issuecomment-1778168339 at 10/24/2023, 03:44 PM PDT
- tonyvang commented on: https://github.com/hackforla/website/issues/5753#issuecomment-1784493339 at 10/29/2023, 10:12 PM PDT
- tonyvang opened: https://github.com/hackforla/website/pull/5794 at 10/29/2023, 10:34 PM PDT
- tonyvang merged: https://github.com/hackforla/website/pull/5794#event-10837630180 at 11/01/2023, 03:46 PM PDT
- tonyvang commented on: https://github.com/hackforla/website/pull/5794#issuecomment-1789840520 at 11/01/2023, 04:43 PM PDT
- tonyvang PR was merged: https://github.com/hackforla/website/pull/5794#event-10850945368 at 11/02/2023, 07:36 PM PDT
- tonyvang issue closed by PR 5794: https://github.com/hackforla/website/issues/5753#event-10850945430 at 11/02/2023, 07:36 PM PDT
- tonyvang assigned: https://github.com/hackforla/website/issues/5876 at 11/12/2023, 09:59 AM PST
- tonyvang commented on: https://github.com/hackforla/website/pull/5887#issuecomment-1808867170 at 11/13/2023, 11:17 AM PST
- tonyvang commented on: https://github.com/hackforla/website/issues/5876#issuecomment-1808867576 at 11/13/2023, 11:18 AM PST
- tonyvang submitted review: https://github.com/hackforla/website/pull/5887#pullrequestreview-1728965237 at 11/13/2023, 09:52 PM PST
- tonyvang issue closed by PR 5917: https://github.com/hackforla/website/issues/5876#event-10960153397 at 11/14/2023, 01:11 PM PST
- tonyvang opened: https://github.com/hackforla/website/pull/5917 at 11/16/2023, 07:39 PM PST
- tonyvang PR was merged: https://github.com/hackforla/website/pull/5917#event-11004749186 at 11/19/2023, 02:24 PM PST
- tonyvang issue closed by PR 5917: https://github.com/hackforla/website/issues/5876#event-11004749228 at 11/19/2023, 02:24 PM PST
- tonyvang commented on: https://github.com/hackforla/website/pull/5915#issuecomment-1819754299 at 11/20/2023, 12:26 PM PST
- tonyvang submitted review: https://github.com/hackforla/website/pull/5915#pullrequestreview-1743610722 at 11/21/2023, 09:52 PM PST
- tonyvang commented on: https://github.com/hackforla/website/issues/5486#issuecomment-1837219837 at 12/02/2023, 10:14 AM PST
- tonyvang commented on: https://github.com/hackforla/website/pull/6081#issuecomment-1884156983 at 01/09/2024, 08:03 PM PST
- tonyvang submitted review: https://github.com/hackforla/website/pull/6103#pullrequestreview-1822437811 at 01/15/2024, 06:30 PM PST
- tonyvang commented on: https://github.com/hackforla/website/pull/6135#issuecomment-1907267865 at 01/23/2024, 06:58 PM PST
- tonyvang submitted review: https://github.com/hackforla/website/pull/6135#pullrequestreview-1844950294 at 01/25/2024, 05:12 PM PST
- tonyvang commented on: https://github.com/hackforla/website/pull/6135#issuecomment-1918298698 at 01/30/2024, 07:04 PM PST
- tonyvang submitted review: https://github.com/hackforla/website/pull/6135#pullrequestreview-1852860814 at 01/30/2024, 08:12 PM PST
- tonyvang assigned: https://github.com/hackforla/website/issues/4931 at 02/05/2024, 06:50 PM PST
- tonyvang commented on: https://github.com/hackforla/website/issues/4931#issuecomment-1928688249 at 02/05/2024, 06:52 PM PST
- tonyvang commented on: https://github.com/hackforla/website/issues/4931#issuecomment-1937823156 at 02/11/2024, 09:57 AM PST
- tonyvang commented on: https://github.com/hackforla/website/pull/6275#issuecomment-1937827381 at 02/11/2024, 10:12 AM PST
- tonyvang unassigned: https://github.com/hackforla/website/issues/4931 at 02/11/2024, 08:26 PM PST
- tonyvang commented on: https://github.com/hackforla/website/pull/6275#issuecomment-1949390574 at 02/16/2024, 01:55 PM PST
- tonyvang assigned: https://github.com/hackforla/website/issues/6310 at 02/20/2024, 06:21 PM PST
- tonyvang commented on: https://github.com/hackforla/website/issues/6310#issuecomment-1955751383 at 02/20/2024, 06:22 PM PST
- tonyvang opened: https://github.com/hackforla/website/pull/6368 at 02/23/2024, 04:57 PM PST
- tonyvang PR was merged: https://github.com/hackforla/website/pull/6368#event-11913592557 at 02/25/2024, 10:32 AM PST
- tonyvang issue closed by PR 6368: https://github.com/hackforla/website/issues/6310#event-11913592589 at 02/25/2024, 10:32 AM PST
- tonyvang commented on: https://github.com/hackforla/website/pull/6384#issuecomment-1968192859 at 02/27/2024, 08:26 PM PST
- tonyvang submitted review: https://github.com/hackforla/website/pull/6384#pullrequestreview-1912390097 at 03/01/2024, 04:36 PM PST
- tonyvang submitted review: https://github.com/hackforla/website/pull/6393#pullrequestreview-1912564817 at 03/01/2024, 07:11 PM PST
- tonyvang commented on: https://github.com/hackforla/website/issues/5486#issuecomment-1974300616 at 03/01/2024, 08:56 PM PST
- tonyvang submitted review: https://github.com/hackforla/website/pull/6384#pullrequestreview-1912582848 at 03/01/2024, 09:37 PM PST
